src/Components/CourseScreen.css

```diff
@@ -47,13 +47,13 @@
   background-color: #F21E1D;
 }
 
-form {
+/* form {
```

Comment:
uncommenting and commenting different options, obviously change for final build, but good for devlopment mid project